src/udev/udev-rules.c

```
@@ -1031,30 +1032,16 @@ static int parse_line(char **line, char **ret_key, char **ret_attr, UdevRuleOper
         key_end[0] = '\0';
 
         tmp += op == OP_ASSIGN ? 1 : 2;
-        value = skip_leading_chars(tmp, NULL);
-
-        /* value must be double quotated */
-        if (value[0] != '"')
+        tmp = skip_leading_chars(tmp, NULL);
+        r = udev_rule_parse_value(tmp, ret_value, &i);
+        if (r < 0) {
                 return -EINVAL;
```

Unnecessary `{}`, also, please propagate the error in `udev_rule_parse_value()`.
```
r = udev_rule_parse_value(...);
if (r < 0)
        return r;
```